features/flows/rest/impl/src/main/java/org/opennms/netmgt/flows/rest/internal/FlowRestServiceImpl.java

```diff
@@ -327,4 +329,26 @@ private static void populateResponseFromTable(Table<?, Long, Double> table, Flow
         response.setValues(values);
     }
 
+    @Override
+    public String getDeepDiveToolUrl(UriInfo uriInfo) {
```

Comment:
We may want to return a `Response` instead of a `String`, as `null` would be converted to a `404`.
I'd prefer a `204` in this case.

Reply:
Actually, this is not true. `null` will return a `204`, so everything is fine.